### DISCUSSION TITLE
Deepstack :: Improving Accuracy

### DISCUSSION THREAD

**Discussion by gniknalu** [roflcoopter/viseron#348] [Q&A]
My deepstack detections could be improved. For example, darknet always got it right when it was a cat vs a dog. "Person" was normally, in darknet, in the high 90 percentile. Deepstack always thinks a cat is a dog.  

![FrontE_2022-05-13-062023](https://user-images.githubusercontent.com/3652809/168310989-a043ad20-e360-4535-ab6a-3234e59439df.jpg)

Started reading articles discovered via a google search on how to improve deepstack accuracy but wanted to see what is pertinent to Viseron when it comes to improving deepstack detection? Any suggestions on what can be done?

Than you!

**Comment by roflcoopter** [comment]
There is not much that can be done on the side of Viseron.
Viseron simply uses the API of Deepstack, so its kind of "black-box"

So any optimization would have to be done on the Deepstack side of things
